userguide/storage.rst

```diff
@@ -713,53 +713,70 @@ existing datasets to close them--the remaining
-once deduplication is enabled**, as disabling deduplication has
-**NO EFFECT** on existing data. The more data written to a
-deduplicated dataset, the more RAM it requires. When the system starts
-storing the DDTs (dedup tables) on disk because they no longer fit
-into RAM, performance craters. Further, importing an unclean pool can
-require between 3-5 GB of RAM per terabyte of deduped data, and if the
-system does not have the needed RAM, it will panic. The only solution
-is to add more RAM or recreate the pool.
-**Think carefully before enabling dedup!**
-This `article
-<http://constantin.glez.de/blog/2011/07/zfs-dedupe-or-not-dedupe>`_
-provides a good description of the value versus cost considerations
-for deduplication.
-
-**Unless a lot of RAM and a lot of duplicate data is available, do not
-change the default deduplication setting of "Off".**
-For performance reasons, consider using compression rather than
-turning this option on.
-
-If deduplication is changed to *On*, duplicate data blocks are removed
-synchronously. The result is that only unique data is stored and
-common components are shared among files. If deduplication is changed
-to *Verify*, ZFS will do a byte-to-byte comparison when two blocks
-have the same signature to make sure that the block contents are
-identical. Since hash collisions are extremely rare, *Verify* is
-usually not worth the performance hit.
-
-.. note:: After deduplication is enabled, the only way to disable it
-   is to use the :samp:`zfs set dedup=off {dataset_name}` command
-   from :ref:`Shell`. However, any data that has already been
-   deduplicated will not be un-deduplicated. Only newly stored data
-   after the property change will not be deduplicated. The only way to
-   remove existing deduplicated data is to copy all of the data off of
-   the dataset, set the property to off, then copy the data back in
-   again. Alternately, create a new dataset with
-   :guilabel:`ZFS Deduplication` left disabled, copy the data to the
-   new dataset, and destroy the original dataset.
+#ifdef freenas
+.. index:: Deduplication
+.. _Deduplication:
+
+Deduplication
+^^^^^^^^^^^^^
+
+Deduplication ("dedup") is a process where ZFS aims to store a single copy
+of any data that is repeated in different blocks or files, rather than storing 
+identical copies each time the data occurs. Deduplication 
+potentially allows repeated and identical data to be stored very 
+efficiently in a smaller space, but also places a 
+very heavy burden on RAM and on the CPU. In particular, dedup 
+significantly reduces write performance and usually requires large (and at 
+times huge) amounts of RAM.  See :ref:`Deduplication RAM requirements` for details.
+Because of the demands that dedup places on a system and the impact on performance, 
+enabling dedup is **usually not recommended** and the setting should be left in
+the default setting ("off") in most situations.
+
```

Comment:
remove "Because of the" Something like:
Due to the demands that deduplication places on a system and the impacts on performance, enabling deduplication is **not recommended**. Deduplication is off by default and should not be enabled without good reason.

Reply:
Done. Reworded slightly, might need review.

Reply:
Done. Reworded slightly, might need review.